chromadb/test/property/test_add.py

```diff
@@ -2,14 +2,17 @@
 import hypothesis.strategies as st
 from hypothesis import given, settings
 from chromadb.api import API
+from chromadb.api.types import Embeddings
 import chromadb.test.property.strategies as strategies
 import chromadb.test.property.invariants as invariants
 
 collection_st = st.shared(strategies.collections(with_hnsw_params=True), key="coll")
 
 
-@given(collection=collection_st, embeddings=strategies.recordsets(collection_st))
-@settings(deadline=None)
+@given(
+    collection=collection_st, record_set=strategies.recordsets(collection_st)
+)  # type: ignore
```

Comment:
These type: ignores can probably go away when we merge @levand's changes from main which add hypothesis to precommit